SMLHelper/Json/ConfigFile.cs

```diff
@@ -33,6 +39,26 @@ public abstract class ConfigFile : IJsonFile
         /// <seealso cref="alwaysIncludedJsonConverters"/>
         public JsonConverter[] AlwaysIncludedJsonConverters => alwaysIncludedJsonConverters;
 
+        /// <summary>
+        /// Creates a new instance of <see cref="ConfigFile"/>, parsing the filename and subfolder from a
+        /// <see cref="ConfigFileAttribute"/> if declared, or with default values otherwise.
+        /// </summary>
+        public ConfigFile()
+        {
+            if (GetType().GetCustomAttributes(typeof(ConfigFileAttribute), true).FirstOrDefault() is ConfigFileAttribute configFile)
```

Comment:
Are we able to use the Generic form of `GetCustomAttributes` here? Or is that not available in this version of .NET?
```CS
GetType().GetCustomAttributes<ConfigFileAttribute>(true)...
```

Reply:
Not available.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.reflection.customattributeextensions.getcustomattributes?view=netcore-3.1&viewFallbackFrom=netframework-4.0#System_Reflection_CustomAttributeExtensions_GetCustomAttributes__1_System_Reflection_MemberInfo_System_Boolean_

Reply:
We are not on .NET Framework 4.7.2, so I've implemented this change :) 5b5fe4e